build/templates/session.py/default_method.py.mako

```diff
@@ -16,4 +19,14 @@
 % for parameter in enum_input_parameters:
         ${helper.get_enum_type_check_snippet(parameter, indent=12)}
 % endfor
-        return self._library_interpreter.${f['library_interpreter_name']}${suffix}(${helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_INTERPRETER_METHOD_CALL)})
+% for p in helper.filter_parameters(f, helper.ParameterUsageOptions.LIBRARY_INTERPRETER_METHOD_CALL):
+%   if 'python_api_converter_name' in p:
+        ${p['python_name']} = _converters.${p['python_api_converter_name']}(${p['python_name']})
+%   endif
+% endfor
+% if output_parameters:
+        ${output_parameters_snippet} = self._library_interpreter.${f['library_interpreter_name']}${suffix}(${helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_INTERPRETER_METHOD_CALL)})
```

Comment:
Can't we just return what's on the right-hand side of this = sign?
Doesn't get_session_method_return_snippet() just result in `return ${output_parameters_snippet}`?

I realize named output parameters makes for better documentation but is it worth all of this extra complexity?
At the very least, it looks like we're arriving at the same list of output parameters in 2 different ways?

Reply:
Often it's the same list, but when there are converters involved, the output_parameters_snippet will invoke those converters on the appropriate output parameters.

I _could_ detect when there are no converters involved, and just do the `return` directly in those cases, but since this is just generated code, I don't think it's worth the added complexity.